auth-ldap/00-setup.sh

```diff
@@ -9,26 +9,38 @@ TEST_INSTANCE_ID=1
 exec_ssoadm $TEST_INSTANCE_ID delete-realm \
   --adminid amadmin \
   --password-file pwd.txt \
-  --realm $TEST_REALM || : # ignore error exit code in case the realm does not exist
+  --realm "$TEST_REALM" \
+> /dev/null || :
 
 # Create new realm
 exec_ssoadm $TEST_INSTANCE_ID create-realm \
   --adminid amadmin \
   --password-file pwd.txt \
-  --realm $TEST_REALM
+  --realm "$TEST_REALM" \
+> /dev/null
 
 # Configure default LDAP module
 exec_ssoadm $TEST_INSTANCE_ID update-auth-instance \
   --adminid amadmin \
   --password-file pwd.txt \
   --name LDAP \
-  --realm $TEST_REALM \
-  --datafile /srv/wrenam/LDAP.properties
+  --realm "$TEST_REALM" \
+  --datafile /srv/wrenam/LDAP.properties \
+> /dev/null
 
 # Configure default ldapService chain
 exec_ssoadm $TEST_INSTANCE_ID update-auth-cfg-entr \
   --adminid amadmin \
   --password-file pwd.txt \
   --name ldapService \
-  --realm $TEST_REALM \
-  --entries "LDAP|REQUISITE|"
+  --realm "$TEST_REALM" \
+  --entries "LDAP|REQUISITE|" \
+> /dev/null
+
+# Make sure test user's password is what we expect
+exec_ldap bash -c "ldapmodify -D $ADMIN_DN -w $ADMIN_PASSWORD <<< '
```

Comment:
This is a strange construct. Can't we drop the bash middleman? The following works just fine:

```bash
docker exec -i wrenam-ldap1 ldapmodify -D cn=admin,dc=example,dc=org -w password -x << END
dn: dc=example,dc=org
changetype: modify
replace: description
description: HELLO
END
```

Reply:
Fixed it in the latest commit. But since I want to pass the ldapmodify command to a function (`exec_ldap`), I can't use heredoc, or I don't know how at least. Is that alright?